flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -149,7 +134,7 @@ Actor with ActorLogMessages with ActorLogging {
 
   override def receiveWithLogMessages: Receive = {
     case RegisterTaskManager(connectionInfo, hardwareInformation, numberOfSlots) =>
-      val taskManager = sender
+      val taskManager = sender()
```

Comment:
This does not work with older Akka versions, which we are using with the Hadoop-2.0.0-alpha profile. I think in older Akka version it is a val.